e2e-olm.sh

```diff
@@ -36,7 +47,7 @@ test::object_assert() {
   local args=${5:-}
 
   for j in $(seq 1 ${tries}); do
-    res=$(eval oc get ${args} ${object} -o jsonpath=\"${request}\")
+    res=$(eval oc get ${args} ${object} -o jsonpath=\"${request}\") || :
```

Comment:
Sorry just for my edification what is this doing?

Reply:
With `set -e`, bash will exit if any command returns an error (a non-zero error code).  Using `|| :` literally means "if the previous process returned an error, execute the command `:` which is the 'no-op' command".  This is how you "catch" exceptions in bash and ignore them.